docs/stdlib/json.rst

```diff
@@ -272,7 +268,7 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
 
     :index: json parse loads
 
-    Return JSON value represented by the input *string*.
+    Returns the value of the inputted :eql:type:`str` from a :eql:type:`json`:
 
     .. code-block:: edgeql-repl
```

Comment:
I think the examples in the code block below muddy the issue. We're trying to show examples of the `to_json` function, but we're also adding an operator to the end of each line which doesn't really add to the example of `to_json`.

Suggestion:
```rest
    .. code-block:: edgeql-repl
        db> select to_json('[1, "hello", null]');
        {"[1, \"hello\", null]"}
        db> select to_json('{"hello": "world"}')['hello'];
        {"{\"hello\": \"world\"}"}
```

@vpetrovykh As I was testing this example, I noticed the actual output of the REPL is this:
```
{Json("[1, \"hello\", null]")}
```
and
```
{Json("{\"hello\": \"world\"}")}
```

I cleaned it up a bit to better match the output in the other REPL blocks, but I'm wondering if we shouldn't be including everything that the REPL spits out in this block in order to avoid surprising the user if they're trying out our examples in their own REPL. If you agree we should, we may need to look at all JSON REPL examples since they would all be printed as `Json(<json here>)` in the REPL.

Reply:
Fixed the first point, but my second question might still be worth looking it.